Blosc/config.py

```diff
@@ -0,0 +1,24 @@
+{
+
+	"downloads" : [
+
+		"https://github.com/Blosc/c-blosc/archive/v1.14.4.tar.gz"
+
+	],
+
+	"license" : "LICENSES/BLOSC.txt",
+
+	"commands" : [
+
+		"cmake -E make_directory {buildDir}/doc/licenses/blosc && cmake -E copy LICENSES/* {buildDir}/doc/licenses/blosc/",
```

Comment:
Interesting - I didn't know about these platform independent commands. I note that below you're not using them though and we're also not using them anywhere else. Is this something we'll need to do consistently everywhere?

Reply:
Yeah I discovered these recently myself, so far I'm only using it because of windows erroring out on a directory that's possibly existing or not and the copy/cp command differences.